src/System.Globalization.Extensions/tests/IdnMapping/IdnMappingGetUnicodeTests.cs

```diff
@@ -9,6 +9,27 @@ namespace System.Globalization.Tests
 {
     public class IdnMappingGetUnicodeTests
     {
+        public static IEnumerable<object[]> GetAscii_TestData()
+        {
+            yield return new object[] { "xn--yda", 0, 7, "\u0101", };
+            yield return new object[] { "axn--ydab", 1, 7, "\u0101" };
+        }
```

Comment:
There are a bunch of inputs in the GetAscii tests in another file.  Could we use those here as well (in the other direction)?